### PR TITLE
Deserializers[] filter operator

### PR DESCRIPTION
**Pull request by saqimtiaz**
This PR adds a `deserializers[]` filter operator to allow users to see available deserializers, as well as docs and a test.

Currently the DropZone widget docs state the following which is far from user friendly: 
> The list of available deserializers can be inspected by executing Object.keys($tw.Wiki.tiddlerDeserializerModules).sort().join("\n") in the browser JavaScript console.

### CODE REVIEW

File: editions/tw5.com/tiddlers/filters/deserializers Operator.tid [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+created: 20210506115203172
+modified: 20210506115607392
+op-input: ignored
+op-output: the title of each available deserializer
+op-parameter: none
+tags: [[Filter Operators]] [[Special Operators]]
+title: deserializers Operator
+type: text/vnd.tiddlywiki
+
+<<.tip "You can specify a specific deserializer for a DropzoneWidget to use">>
```
Comment (pmario) [inline editions/tw5.com/tiddlers/filters/deserializers Operator.tid:11]: `<<.from-version "5.1.24">>`  is missing somewhere, where it looks good

Comment (saqimtiaz) [reply editions/tw5.com/tiddlers/filters/deserializers Operator.tid:12]: Added, thanks @pmario

**Comment by saqimtiaz** [comment]
Conflicts resolved, this is good to go.

**Comment by Jermolene** [comment]
Thanks @saqimtiaz

**Pull request merged**